src/js/tech/html5.js

```diff
@@ -105,29 +91,29 @@ class Html5 extends Tech {
         const clone = el.cloneNode(false);
         Html5.disposeMediaElement(el);
         el = clone;
-        player.tag = null;
+        // Is it really necessary?
+        //player.tag = null;
```

Comment:
I don't remember the specifics, but it is necessary. We should probably just null it when loading the first tech, or find a way to not store it in the first place.